common/scylla_configure.py

```diff
@@ -60,8 +60,6 @@ class ScyllaMachineImageConfigurator:
         'raid_level': 0  # Default raid level is 0, supported raid 0, 5
     }
 
-    DISABLE_START_FILE_PATH = Path("/etc/scylla/ami_disabled")
```

Comment:
now I've remembered in something else, the cloud team was using this file to wait for scylla machine image finish it's boot.

you'll need to cross check with them if they are still using this file as the indicator, or moved to using a different method

Reply:
it's actually a different file:
https://github.com/scylladb/siren/blob/f0c2ffae40618e395ec2a196219a54c6207b4bb2/cluster/server/instance.go#L28

so scratch that...